src/content/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-9-4-0.mdx

```diff
@@ -0,0 +1,37 @@
+---
+subject: Node.js agent
+releaseDate: '2022-10-24'
+version: 9.4.0
+downloadLink: 'https://www.npmjs.com/package/newrelic'
+---
+
+## Notes
+
+* Removed legacy agent async context propagation. The default behavior is now what was behind the `feature_flag.new_promise_tracking`. You can read more about the difference [here](/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-7-3-0/#new-features). 
+
+* Fixed an issue with the ES Module loader that properly registers instrumentation when the file path included url encoded characters.
+
+* Added an API for enqueuing application logs for forwarding 
+
+    ```js
+    newrelic.recordLogEvent({ message: 'hello world', level: 'info' })`
+    ```
+
+    **Note**: If you're including a serialized error make sure it's on the `error` key of the log event: 
+
+    ```js
+    const error = new Error('testing errors'); 
+    newrelic.recordLogEvent({ message: 'error example', level: 'error', error })
+    ```
+
+* Fixed `cassandra-driver` instrumentation to properly set instance details on query segments/spans.
+
+* Added a new context manager that leverages AsyncLocalStorage for async context propagation.
+    * This will be available via a feature flag `config.feature_flag.async_local_context`
+    * Alternatively you can set the environment variable of `NEW_RELIC_FEATURE_FLAG_ASYNC_LOCAL_CONTEXT=1`
+    * By enabling this feature flag it should make the agent use less memory and CPU.
+
+
+### Support statement:
+
+* New Relic recommends that you upgrade the agent regularly to ensure that you're getting the latest features and performance benefits. Additionally, older releases will no longer be supported when they reach [end-of-life](/docs/licenses/end-of-life/notification-changes-new-relic-saas-features-distributed-software/).
```

Comment:
i commented on the commit itself but this is automated on our end. it appears the old url redirects. should we update our automation though to point to this new URL?

Reply:
No, it's not necessary. I did it because I was editing it, but the URL works, so it doesn't matter. Thank you for the update!!!